### PR TITLE
Changed "Manual" string to "autoBattle|Manual" to separate it from game credits "Manual".

### PR DESCRIPTION
**Pull request by Districh-ru**
The word "Manual" has different meanings and is translated differently for the "game manual" and for "manual battales".

For example, it could not be correctly translated into Russian:
- for autobattles it should be translated: "Вручную";
- for game titles it should be translated: "Руководство по игре".

So, now in game credits there is incorrect translation:
изображение
But it cannot be fixed in the translation file. Because this string ("Manual") is also used in autobattles settings dialog:
изображение изображение

This PR replaces "Manual" with "autoBattle|Manual" in the 'dialog_system_options.cpp' and 'dialog_game_settings.cpp' files.

PS
This is my first PR for the game code, so I hope for your help with it. :)

### CODE REVIEW

**Review by zenseii — CHANGES_REQUESTED** [review]
Hi @Districh-ru.

The changes look good to me, however I believe we should add a context to the "Manual" string in the credits too then. Something like "credits|Manual", because this will remove any ambiguity.

**Review by zenseii — APPROVED** [review]

**Comment by Districh-ru** [comment]
Hi @zenseii 
I did it. The string "Manual" is only in these three files.

**Comment by zenseii** [comment]
> Hi @zenseii I did it. The string "Manual" is only in these three files.

Yep, thanks! I should also note that technically the engine tries to differentiate between "auto battle" and "auto resolve" of battles, in which the former refers to entering a battle and giving the computer the control of the moves and watching these take place, while the latter is the one where the battle is automatically solved before even entering the battle screen.

However, this terminology is admittedly not very clear so to me "autoBattle", which is also what I suggested in the other PR if I recall correctly, works perfectly fine and will be understood correctly by the translator.

**Comment by Districh-ru** [comment]
> However, this terminology is admittedly not very clear so to me "autoBattle", which is also what I suggested in the other PR if I recall correctly, works perfectly fine and will be understood correctly by the translator.

So maybe change it to "battle|Manual", because "Manual" string is used for battle configuration. (The other two settings for Battle is "Auto Resolve" and "Auto, No Spells").

**Comment by zenseii** [comment]
> > However, this terminology is admittedly not very clear so to me "autoBattle", which is also what I suggested in the other PR if I recall correctly, works perfectly fine and will be understood correctly by the translator.
> 
> So maybe change it to "battle|Manual" 

I'm not sure this is any better because you still have ambiguity which type of automatic battle this is talking about. While "Auto Battle" is a term most can understand. I don't think there's much need to spend more time figuring out what to call these things right now because the context you provided works perfectly fine, as I said.


> (The other two settings for Battle is "Auto Resolve" and "Auto, No Spells").

I believe "Auto, No spells" is just short for "Auto Resolve, No Spells" because the space is limited, so really as far as I've understood "Auto Battle" is technically when you click the "Auto" button in the lower left corner in the battle screen, while "Auto Resolve" is when you don't even enter the battle screen. My point is also that a context like "autoResolve" is too vague for a translator to easily understand.

In later games in the series Auto Resolve is referred to as "Quick Combat" and "Auto Battle" is exactly what I described it as above. However, we have not adopted these terms in fheroes2.  At least not yet.

**Comment by Districh-ru** [comment]
@zenseii 
Thanks for the detailed explanation! :)

**Comment by ihhub** [comment]
Hi @Districh-ru and @zenseii , we need to change translation files as well because we modified the original English sentence.

**Comment by Districh-ru** [comment]
Hi @ihhub , ok, I'll do it today in the evening (after 9 hours).
I thought that translation files will be modified automatically)

**Comment by zenseii** [comment]
> Hi @Districh-ru and @zenseii , we need to change translation files as well because we modified the original English sentence.

We would need to ask the translators for help for that because we don't know what is the correct translation for every language here.

**Comment by zenseii** [comment]
@Districh-ru, I made an update to my comment.

**Comment by Districh-ru** [comment]
@zenseii, thanks for the tip!
I installed and configured Cygwin, made a POT file and updated the PO files for translation.
I made a Russian translation and also tried to keep the correct translation for languages where "Manual" has already been translated.

File: files/lang/nb.po [hunk 1]
```diff
@@ -3602,6 +3602,10 @@ msgstr "Sopran"
 msgid "Recorded at Green Street Studios"
 msgstr "Opptak gjort ved Green Street Studios"
 
+#, fuzzy
+msgid "credits|Manual"
+msgstr "Guide"
```
Comment (zenseii) [inline files/lang/nb.po:3607]:
```suggestion
msgstr "Håndbok"
```

File: files/lang/nb.po [hunk 2]
```diff
@@ -2760,7 +2760,7 @@ msgstr "Datastyrt"
 msgid "Auto, No Spells"
 msgstr "Data, ingen magi"
 
-msgid "Manual"
+msgid "autoBattle|Manual"
 msgstr "Manuelt"
```
Comment (zenseii) [inline files/lang/nb.po:2764]:
```suggestion
msgstr "Manuell"
```

File: files/lang/it.po [hunk 3]
```diff
@@ -2596,8 +2596,9 @@ msgstr "Auto-Risolva"
 msgid "Auto, No Spells"
 msgstr "Auto, Senza Magie"
 
-msgid "Manual"
-msgstr "Manuali"
+#, fuzzy
+msgid "autoBattle|Manual"
+msgstr "Manualmente"
```
Comment (zenseii) [inline files/lang/it.po:2601]:
```suggestion
msgstr "Manuali"
```

File: files/lang/it.po [hunk 4]
```diff
@@ -2596,8 +2596,9 @@ msgstr "Auto-Risolva"
 msgid "Auto, No Spells"
 msgstr "Auto, Senza Magie"
 
-msgid "Manual"
-msgstr "Manuali"
+#, fuzzy
```
Comment (Districh-ru) [inline files/lang/it.po:2599]:
```suggestion

```

File: files/lang/nb.po (hunk 1 above)
Comment (Districh-ru) [inline files/lang/nb.po:3607]:
```suggestion
msgid "credits|Manual"
msgstr "Håndbok"
```

Comment (Districh-ru) [reply files/lang/nb.po]: @zenseii, thanks for correcting translations!
I applied your suggestions.

Comment (Districh-ru) [reply files/lang/nb.po]: This suggestion didn't show up in the Files Changed tab (probably a Github bug), so I made my suggestion with those changes.

File: files/lang/fr.po [hunk 5]
```diff
@@ -2733,7 +2733,8 @@ msgstr "Combat auto"
 msgid "Auto, No Spells"
 msgstr "Auto sans magie"
 
-msgid "Manual"
+#, fuzzy
+msgid "autoBattle|Manual"
 msgstr "Manuel"
```
Comment (zenseii) [inline files/lang/fr.po:2738]: Please remove the fuzzy too, @Districh-ru .
```suggestion
msgstr "Manuelles"
```

File: files/lang/fr.po [hunk 6]
```diff
@@ -3567,6 +3568,10 @@ msgstr "Soprano (Voix)"
 msgid "Recorded at Green Street Studios"
 msgstr "Enregistrée aux Green Street Studios"
 
+#, fuzzy
+msgid "credits|Manual"
+msgstr "Guide"
```
Comment (zenseii) [inline files/lang/fr.po:3573]: Hi @1GodRage and @LeHerosInconnu.
According to the dictionary this translation works,
But I feel something more akin to Handbook is better.

File: files/lang/it.po [hunk 7]
```diff
@@ -3432,6 +3432,10 @@ msgstr ""
 msgid "Recorded at Green Street Studios"
 msgstr ""
 
+#, fuzzy
+msgid "credits|Manual"
+msgstr "Manuali"
```
Comment (zenseii) [inline files/lang/it.po:3437]: Please remove fuzzy mark too, @Districh-ru 
```suggestion
msgstr "Manuale"
```

File: files/lang/pl.po [hunk 8]
```diff
@@ -2700,8 +2700,9 @@ msgstr "Automatyczna"
 msgid "Auto, No Spells"
 msgstr "Auto, bez zaklęć"
 
-msgid "Manual"
-msgstr "Ręczna"
+#, fuzzy
+msgid "autoBattle|Manual"
+msgstr "Ręcznie"
```
Comment (zenseii) [inline files/lang/pl.po:2705]: Hi @Arthusppp, @Pulfiarz and @KaspianDev.

I would like to ask your help with these two strings if you have the time of course.

**Review by zenseii — CHANGES_REQUESTED** [review]
I requested some help from some of our contributors to the actively maintained translations, @Districh-ru . Hopefully they will have the time to help out.

I've already covered the languages I can help out with.

File: files/lang/fr.po [hunk 9]
```diff
@@ -2733,7 +2733,8 @@ msgstr "Combat auto"
 msgid "Auto, No Spells"
 msgstr "Auto sans magie"
 
-msgid "Manual"
+#, fuzzy
```
Comment (Districh-ru) [inline files/lang/fr.po:2736]:
```suggestion
```

File: files/lang/fr.po (hunk 5 above)
Comment (Districh-ru) [reply files/lang/fr.po:2738]: OK

File: files/lang/it.po [hunk 10]
```diff
@@ -3432,6 +3432,10 @@ msgstr ""
 msgid "Recorded at Green Street Studios"
 msgstr ""
 
+#, fuzzy
```
Comment (Districh-ru) [inline files/lang/it.po:3435]:
```suggestion
```

File: files/lang/it.po (hunk 7 above)
Comment (Districh-ru) [reply files/lang/it.po:3437]: OK

File: files/lang/fr.po (hunk 6 above)
Comment (zenseii) [reply files/lang/fr.po:3573]: Also if you have some ideas about how to translate this best too, @morgane-du-tweize , then please share 👍

File: files/lang/bg.po [hunk 11]
```diff
@@ -2701,7 +2701,7 @@ msgstr "Авто"
 msgid "Auto, No Spells"
 msgstr "Авто, без магий"
 
-msgid "Manual"
+msgid "autoBattle|Manual"
```
[the remark this answers is not in the document]
Comment (Effektus) [reply files/lang/bg.po:2704]: автоматичната битка/Ръчно
or
авто битка/Ръчно

File: files/lang/bg.po [hunk 12]
```diff
@@ -3527,6 +3527,10 @@ msgstr "Сопрано вокал"
 msgid "Recorded at Green Street Studios"
 msgstr "Записано в Green Street Studios"
 
+#, fuzzy
+msgid "credits|Manual"
+msgstr "Ръководство за играта"
```
[the remark this answers is not in the document]
Comment (Effektus) [reply files/lang/bg.po:3532]: Руководство по игре

[the remark this answers is not in the document]
Comment (Districh-ru) [reply files/lang/bg.po:3532]:
```suggestion
msgstr "Руководство по игре"
```
Thank you, @Effektus !

File: files/lang/bg.po [hunk 13]
```diff
@@ -3527,6 +3527,10 @@ msgstr "Сопрано вокал"
 msgid "Recorded at Green Street Studios"
 msgstr "Записано в Green Street Studios"
 
+#, fuzzy
```
Comment (Districh-ru) [inline files/lang/bg.po:3530]:
```suggestion
```

File: files/lang/bg.po [hunk 14]
```diff
@@ -2701,7 +2701,7 @@ msgstr "Авто"
 msgid "Auto, No Spells"
 msgstr "Авто, без магий"
 
-msgid "Manual"
+msgid "autoBattle|Manual"
 msgstr "Ръчно"
```
Comment (Districh-ru) [inline files/lang/bg.po:2705]:
```suggestion
msgstr "Вручную"
```

File: files/lang/fr.po (hunk 6 above)
Comment (morgane-du-tweize) [reply files/lang/fr.po:3573]: It depends on the context.
Basically : "manuel" : basic instructions. Tells how to play. Usually given with the game when you buy a physical copy (well it was like that in the 90's)
On steam, when browsing a game hub : guides tells you how to win (wakthroughs, achievements...) and it's directly translated to "guide" in french.

Comment (zenseii) [reply files/lang/fr.po:3573]: Thanks @morgane-du-tweize.
I vote for "Manuel" here in this case.

Comment (Districh-ru) [reply files/lang/fr.po:3573]: Thanks @morgane-du-tweize and @zenseii !
I'll also remove the fuzy mark.
```suggestion
msgstr "Manuel"
```

File: files/lang/fr.po [hunk 15]
```diff
@@ -3567,6 +3568,10 @@ msgstr "Soprano (Voix)"
 msgid "Recorded at Green Street Studios"
 msgstr "Enregistrée aux Green Street Studios"
 
+#, fuzzy
```
Comment (Districh-ru) [inline files/lang/fr.po:3571]:
```suggestion
```

File: files/lang/ru.po [hunk 16]
```diff
@@ -9,6 +9,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: fheroes2\n"
 "Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-11-01 18:49+0300\n"
```
Comment (Districh-ru) [inline files/lang/ru.po:12]: Removed duplicate line
```suggestion
```

File: files/lang/bg.po (hunk 11 above)
[the remark this answers is not in the document]
Comment (Districh-ru) [reply files/lang/bg.po:2704]: The suggestion has applied. Thank you.

File: files/lang/pl.po (hunk 8 above)
Comment (Arthusppp) [reply files/lang/pl.po:2705]: I suggest "Ręczna" here.

File: files/lang/pl.po [hunk 17]
```diff
@@ -3521,6 +3522,10 @@ msgstr "Sopran (partie wokalne)"
 msgid "Recorded at Green Street Studios"
 msgstr "Nagrano w Green Street Studios"
 
+#, fuzzy
+msgid "credits|Manual"
+msgstr "Przewodnik"
```
Comment (Arthusppp) [inline files/lang/pl.po:3527]: This is ok, please remove "#, fuzzy".

File: files/lang/pl.po (hunk 8 above)
Comment (Districh-ru) [reply files/lang/pl.po:2705]:
```suggestion
msgstr "Ręczna"
```

File: files/lang/pl.po [hunk 18]
```diff
@@ -2700,8 +2700,9 @@ msgstr "Automatyczna"
 msgid "Auto, No Spells"
 msgstr "Auto, bez zaklęć"
 
-msgid "Manual"
-msgstr "Ręczna"
+#, fuzzy
```
Comment (Districh-ru) [inline files/lang/pl.po:2703]:
```suggestion
```

File: files/lang/pl.po (hunk 17 above)
Comment (Districh-ru) [reply files/lang/pl.po:3527]: Hi @Arthusppp, thank you!

File: files/lang/pl.po [hunk 19]
```diff
@@ -3521,6 +3522,10 @@ msgstr "Sopran (partie wokalne)"
 msgid "Recorded at Green Street Studios"
 msgstr "Nagrano w Green Street Studios"
 
+#, fuzzy
```
Comment (Districh-ru) [inline files/lang/pl.po:3525]:
```suggestion
```

**Comment by Districh-ru** [comment]
@zenseii, I applied the translators suggestions.

**Review by zenseii — APPROVED** [review]
All actively maintained translations have been covered.
This PR is ready for you to have a look at it when you have the time, @ihhub.

**Comment by ihhub** [comment]
@Districh-ru , thanks a lot for this improvement!

**Pull request merged**